### PR TITLE
rm package.json preferGlobal:true

### PR DESCRIPTION
**Pull request by isao**
and add deprecation notice about installing mojito globally.

### CODE REVIEW

**Comment by isao** [comment]
@caridy if mojito-cli will ever be installed locally in an app (as opposed to globally) as "mojito", then we also need to change or remove the "bin": {"mojito": "bin/mojito"} in package.json as well.

**Comment by caridy** [comment]
@isao yes, I don't think we will need the bin in both places once the cli gets ready, and we clean up mojito.

File: HISTORY.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ Here are details of the API changes.
 
 Please see the API docs for details of each.
 
+Installing mojito globally (i.e. `npm install --global mojito`) is deprecated. Please install mojito locally in your application.
```
Comment (caridy) [inline HISTORY.md]: I will suggest that we don't call it deprecated that fast. We can suggest that the change is intent to clear the space for the upcoming mojito-cli, but using the current global mojito will help with the archetypes and tests. I don't know, it sounds like to early to push them not to install it globally, when that's what they are used to.

**Comment by caridy** [comment]
+1

**Pull request merged**